sql/core/src/test/scala/org/apache/spark/sql/streaming/DataFrameReaderWriterSuite.scala

```diff
@@ -368,4 +368,79 @@ class DataFrameReaderWriterSuite extends StreamTest with SharedSQLContext with B
       "org.apache.spark.sql.streaming.test",
       Map.empty)
   }
+
+  test("check continuous query methods should not be called on non-continuous queries") {
```

Comment:
Can you break these up into separate `test(...)`s?  Its helpful when the failure output is clear about what specific function is broken.